docs/CMakeLists.txt

```diff
@@ -53,3 +53,9 @@ compile_documentation()
 compile_test_documentation(
     "${PROJECT_SOURCE_DIR}/test" # Test directory
 )
+
```

Comment:
I am not sure we should install this project when there is nothing installed from it.
I would fix the warning by checking if DOCS_BUILD (or whatever it is called) is set, and if not, do nothing so the project does not get compiled (or even `project` call is not done)

Reply:
That was my first attempt, couldn't make it work.